src/components/stepforms/schemas/rollup.ts

```diff
@@ -35,7 +35,7 @@ export default {
       },
     },
     labelCol: {
-      type: 'string',
+      type: ['null', 'string'],
```

Comment:
@alice-sevin I'm wondering if sending `labelCol: null` will be something that Pydantic's will validate.
... I fear not :/ Maybe the problem is that the widget should set the value to undefined rather than to null if nothing is in the test input?

Reply:
I take a look :pray:

Reply:
Et merde, ça fixe pas le soucis quand on réupdate un champ :cry: , c'est galère, du coup à ce moment là on se retrouve à avoir "" et on a un minLength dans le schema. D'ailleurs je comprends pas ce minLength du coup on pourrait pas simplement le supprimer? Ca fait pas sens pour moi

Reply:
Est-ce que le soucis ne viendrait pas du InputTextWidget?
Si, quand le champ de texte est vide, il émet `undefined` plutôt que `""` ou `null` alors on aurait bien un objet cohérent avec le schéma, non?